src/plugins/data_view_management/public/components/empty_index_list_prompt/empty_index_list_prompt.tsx

```diff
@@ -46,13 +44,13 @@ export const EmptyIndexListPrompt = ({
   const createAnywayLink = (
     <EuiText color="subdued" textAlign="center" size="xs">
       <FormattedMessage
-        id="indexPatternEditor.createIndexPattern.emptyState.createAnywayTxt"
+        id="indexPatternManagement.createDataView.emptyState.createAnywayTxt"
```

Comment:
What's the reasoning for renaming `indexPatternEditor` to `indexPatternManagement`, instead of `dataViewEditor`? Or something that replaces `indexPattern` with `dataView`?

Reply:
Thats the namespace defined for data_view_management where the empty_index_list_prompt is moved to from data_view_editor.